src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -188,435 +188,730 @@ If a problem occurs, check output in /var/log/proactive/scheduler
+[source,yaml]
 ----
+include::./references/automatedInstallScripts/.env[]
+----
+====
+'''
 
+===== Required configuration
+The `.env` file referenced above can be used as it is for a minimal installation. However, it requires the definition of five mandatory parameter values:
 
+- `DOCKER_IMAGE_PROACTIVE_NAMESPACE`: Defines the ProActive Docker image namespace (or repository) in the Activeeon private registry.
+- `DOCKER_IMAGE_PROACTIVE_TAG`: Defines the ProActive release version.
+- `DOCKER_IMAGE_POSTGRES_NAMESPACE`: Defines the ProActive Postgres namespace.
+- `DOCKER_IMAGE_POSTGRES_TAG`: Defines the Postgres version to be used.
+- `PWS_HOST_ADDRESS`: Defines the internal hostname or IP address of the machine where ProActive will be installed.
 
-===== Set up ProActive using PostgreSQL Database
+[[_compose_advanced_parameters]]
+===== Advanced configuration
+For more advanced configuration of the ProActive server, please refer to the parameters listed below:
 
-This Docker Compose configuration `docker-compose-postgres.yml` can be used to run the ProActive server container with an external PostgreSQL Database container.
-It will start up to three containers, one for the server, one for the Nodes, and one for the Database.
+[[_compose_PWS_Server_Configuration_Properties]]
+*_PWS Server Configuration Properties_*
 
-[source]
+This section outlines the global configuration options available to configure the ProActive server.
+
+- `PWS_PROTOCOL`: The internal http protocol to use (http or https). By default, http.
+
+- `PWS_PORT`: The internal port used (e.g., 8080 for http, 8443 for https). By default, 8080.
+
+- `PWS_PAMR_PORT`: The internal port to use for PARM communication. By default, 33647.
+
+- `PWS_JOB_CLEANUP_DAYS`: Remove jobs history older than (in days). By default, 30.
+
+- `PWS_USER_ID`: ID of the user starting the ProActive server. By default, 1001.
+
+- `PWS_USER_GROUP_ID`: Group ID of the user starting the ProActive server. By default, 1001.
+
+- `PWS_USER_NAME`: Name of the user starting the ProActive server. By default, activeeon.
+
+- `PWS_USER_GROUP_NAME`: Group of the user starting the ProActive server. By default, activeeon.
+
+- `PWS_LOGIN_ADMIN_PASSWORD`: Password of the ProActive 'admin' user. By default, activeeon.
+
+- `PWS_WORKER_NODES`: Number of ProActive agent nodes to deploy on the ProActive server machine. By default, 4.
+
+[[_compose_Public_Access_Configuration_Properties]]
+*_Public Access Configuration Properties_*
+
+This section describes the public REST Urls to be set when you are running the ProActive server behind a reverse proxy or in a cloud instance (e.g., Azure VM)
+
+- `ENABLE_PUBLIC_ACCESS`: Set to true when the ProActive server is deployed behind a reverse proxy or inside a cloud instance. By default, false.
+
+- `PWS_PUBLIC_HOSTNAME`: Public hostname used (e.g. public.myserver.mydomain) or IP address (e.g. 192.168.12.1).
+
```

Comment:
192.168.12.1 isn't a private IP address?

Reply:
yeah I got this from the documentation of the "configure-http" command created by @fviale